recipe/meta.yaml

```
@@ -6,21 +6,21 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/langchain_text_splitters-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-','_') }}-{{ version }}.tar.gz
   sha256: ac459fa98799f5117ad5425a9330b21961321e30bc19a2a2f9f761ddadd62aa1
 
 build:
-  noarch: python
+  skip: true  # [py<38]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
 
 requirements:
   host:
-    - python >=3.8,<4.0
+    - python
     - poetry-core >=1.0.0
     - pip
   run:
-    - python >=3.8.1,<4.0
+    - python
     - langchain-core >=0.1.28,<0.2.0
   run_constrained:
     - lxml >=5.1.0,<6.0.0
```

```suggestion
    - lxml >=5.1.0,<6.0.0
    - beautifulsoup4 >=4.12.3,<5
```
https://github.com/langchain-ai/langchain/blob/langchain-core%3D%3D0.1.52/libs/text-splitters/pyproject.toml#L15